### PR TITLE
AfterEffects: refactored integrate doesnt work formulti frame publishes

### PR DESCRIPTION
**Pull request by kalisp**
## Brief description
Frame pattern used in new integrate is not catching frames pattern in AE

## Description
Locally rendered files are renamed into frame pattern that integrator plugin understands.

## Additional info
~~(Added Kuba and Ondrej as reviewers to only highlight this.)
Added `render.farm` for AE into Settings to skip integrating. New publisher should have 'render' as a primary family (and 'render.farm` in families), which would trigger integrating (legacy integrator has 'render.farm' in `exclude_families` , this functionality is currently missing in refactored integrator).~~

~~I think we should keep `skip_host_families` not as a temporary solution to broken publishes for new integrator, but use it instead of 'exclude_families' which might be always useful.~~

## Testing notes:
1. Publish locally in AE
2. Publish on farm in AE

### CODE REVIEW

File: openpype/plugins/publish/integrate.py [hunk 1]
```diff
@@ -167,7 +167,7 @@ class IntegrateAsset(pyblish.api.InstancePlugin):
     skip_host_families = []
 
     def process(self, instance):
-        if self._temp_skip_instance_by_settings(instance):
+        if self.skip_instance_by_settings(instance):
```
Comment (iLLiCiTiT) [inline openpype/plugins/publish/integrate.py:170]: why the rename?

**Comment by iLLiCiTiT** [comment]
> I think we should keep skip_host_families not as a temporary solution to broken publishes for new integrator, but use it instead of 'exclude_families' which might be always useful.

That was removed on purpose and was replaced by having `instance.data["farm"] = True` (you've added it...). If it should be added back then it should not be in settings but handled on purpose. It was removed to reduce responsibility of the integrator.

**Comment by iLLiCiTiT** [comment]
With this PR https://github.com/pypeclub/OpenPype/pull/3611 this PR is not needed

**Review by 64qam — APPROVED** [review]

**Comment by kalisp** [comment]
This PR is still useful for new testing class and for farm publishes.

**Pull request merged**